arcade/examples/text_loc_example_done.py

```diff
@@ -56,7 +56,11 @@ def on_draw(self):
         start_y = 450
         arcade.draw_point(start_x, start_y, arcade.color.BLUE, 5)
         arcade.draw_text(
-            _("Simple line of text in 12 point"), start_x, start_y, arcade.color.BLACK, 12
+            _("Simple line of text in 12 point"),
```

Comment:
Although this change makes sense, why is it in this PR? I thought this was supposed to be sounds-specific.